app/presenters/account_show_presenter.rb

```diff
@@ -135,6 +137,10 @@ def connected_apps
     user.connected_apps.includes([:service_provider_record, :email_address])
   end
 
+  def show_change_option
```

Comment:
Not quite straightforward with the logic yet. I want to show the change option if there is a request for all emails, so there will be a `!` statement somewhere, so that will probably mean that the method name should be changed.